compiler/src/iree/compiler/Dialect/HAL/Transforms/Passes.cpp

```diff
@@ -227,8 +227,9 @@ void buildHALTransformPassPipeline(OpPassManager &passManager,
     // Executable translation
     //----------------------------------------------------------------------------
 
-    FunctionLikeNest(passManager)
-        .addPass(createCPUMaterializeUpperBoundTileSizePass);
+    FunctionLikeNest(passManager).addPass([]() {
```

Comment:
revert?

Reply:
I can't revert it because the method needs an optional input argument.

```
error: too few arguments to function call, expected 1, have 0
    std::get<index>(nestedPassManagers)->addPass(constructor());
```

Reply:
I thought there was a better way to do this, but can't see through the template programming just now.